src/test/java/org/mule/modules/certification/automation/functional/MethodATestCases.java

```diff
@@ -0,0 +1,4 @@
+package org.mule.modules.certification.automation.functional;
+
+public class MethodATestCases {
```

Comment:
Let's add some Javadoc for these classes (basically, they're "data" for some (which?) tests that need to find their data in src/test/java)